content/blog/pulumi-in-a-cloud-native-world/index.md

```diff
@@ -0,0 +1,500 @@
+
+* Integration & Delivery. Automations to version control, test, trace, and deploy all infrastructure from resources,
+  configurations, environments, and secrets as well as orchestration automations to manage provisioning workflows.
+
+* Monitoring & Logging. Components to log, monitor, and observe all infrastructure for greater operational control as
+  well as optimize against unnecessary costs.
+
+* Security & Identity. Security and compliance guardrails that regulate every piece of infrastructure from policies to
+  fine-grained access controls to secrets.
+
+* Resources. Providers that support modern cloud architectures such as Kubernetes, containers, serverless, generative
+  AI, machine learning, data lakes, hybrid cloud/on-premises, and more.
+
+By implementing an IDP, organizations can significantly reduce cognitive load on developers, accelerate time-to-market,
+improve security and compliance, and foster a culture of innovation. IDPs act as a force multiplier, allowing
+development teams to focus on creating business value while the platform handles the underlying complexity of
+cloud-native infrastructure.
+
+The [Cloud Native Computing Foundation](https://www.cncf.io/) (CNCF) landscape offers a large ecosystem of tools and
+technologies that can be leveraged to build these platforms from scratch. However, the sheer number of options can be
+overwhelming, making it difficult for organizations to determine the best combination of tools for their specific needs.
+
+At Pulumi's Customer Experience Team, we've observed that adopting Infrastructure as Code often serves as a catalyst for
+organizations to reassess their entire software delivery process. This reassessment frequently leads to the
+implementation of more streamlined, automated, and secure methodologies – the core principles of platform engineering
+mentioned above.
+
+To provide guidance for our customers, we have created a reference architecture called the `KEBAP` stack, this stack
+offers a structured approach to integrate these technologies into an effective platform engineering solution
+using [Pulumi](/).
+
+## The KEBAP Stack Reference Architecture
+
+The `KEBAP` stack consists of the following (most commonly used) CNCF projects:
+
+- **K**ubernetes (with Kyverno as a bonus)
+- **E**xternal Secrets Operator
+- **B**ackstage
+- **A**rgo CD
+- **P**ulumi
+
+Have a look on the high-level architectural diagram of the `KEBAP` stack to get a first idea of what we are going
+to talk about in the following sections:
+
+<img src="base.png">
+
+## Breaking Down the KEBAP Stack
+
+Let's take a closer look at each component of the `KEBAP` stack and understand what their role. The `KEBAP` is layered
+in such a way that each component builds on the capabilities of the previous one. The five layers of the `KEBAP` stack
+are designed to address different capabilities of our internal developer platform:
+
+### Pulumi
```

Comment:
why did we try to explain Pulumi first. I think you need to have Kubernetes first saying its the control plane and then to talk about Pulumi running inside an operator

Reply:
moved Pulumi section under the Kubernetes section